Service/src/main/java/ticket/Ticket.java

```diff
@@ -0,0 +1,253 @@
+package ticket;
```

Comment:
You have the same packages from capital and lowercase letter, leave only that is started from lowercase

Reply:
put ticket and user packages into 'model' 
we're usually separate by functionality and only than by domain area